public/assets/css/app.css

```diff
@@ -1,691 +1,109 @@
+h1 {
     text-transform: uppercase;
+    font-size:      2em;
+    line-height:    2em;
+    width:          100%;
+    text-align:     center;
+    font-weight:    bold;
 }
-.post_parent {
-    color: #888;
-}
-.post_author {
-    color: #486;
-    text-transform: capitalize;
-    font-weight: bold;
-    margin-bottom: 0.25em;
-}
-.post_tripcode {
-    color: #6A6A6A;
-}
-.post_content{
-    text-align: left;
-}
-.post_content p {
-    overflow: auto;
-    margin-bottom: .375em;
-}
-.post_content_cutted {
-    overflow: hidden;
-    height: 450px;
-}
-.post_expand_btn{
-    color: #6A6A6A;
-    border: 1px dashed #6A6A6A;
-    text-align: center;
-    font-size: .75em;
-    font-weight: bold;
+
+h2 {
     text-transform: uppercase;
-    padding: .5em;
-    margin: .375em 0;
-    cursor: pointer;
-}
-.post_media {
-    width: 200px;
-    float: left;
-    margin-right: 0.75em;
+    font-size:      1.75em;
+    line-height:    1.75em;
+    width:          100%;
+    text-align:     center;
+    font-weight:    bold;
 }
-.post_media img{
-    border: 0.0625em solid #DADADA;
-}
-.media_tag {
-    position: absolute;
-    z-index: 100;
-    background-color: #333;
-    color: #FEFEFE;
+
+h3 {
+    font-size:   1.625em;
+    line-height: 1.625em;
+    width:       100%;
+    text-align:  center;
     font-weight: bold;
-    text-transform: uppercase;
-    font-size: 1.5em;
-    padding: 0.5em;
-    line-height: 1em;
-    border: 1px solid #333;
-    border-radius: 0.125em;
-    margin: 32px 0 0 10px;
-    opacity: .825;
-}
-.post_filename{
-    color: #6A6A6A;
-    font-size: 0.75em;
-    text-align: left;
-    display: inline-block;
-}
-.media_open {
-    color: #333;
-    font-size: 0.625em;
-    text-decoration: underline;
-    text-transform: uppercase;
-    width: 100%;
-    text-align: center;
-    line-height: 2em;
-    display: block;
-}
-.spoiler {
-    filter: blur(2px);
-    color: #6A6A6A;
-    padding: 0 .5em;
-    cursor: pointer;
-}
-.spoiler:hover {
-    filter: blur(0);
-    color: #333;
-    cursor: default;
-}
-.utag {
-    text-decoration: underline;
-}
-.post_additional_actions{
-    display: none;
-}
-.related_posts{
-    padding: 0 0 0 4em;
 }
-.pagination{
-    width: 100%;
-    text-align: center;
+
+h4 {
+    font-size:   1.5em;
+    line-height: 1.5em;
+    width:       100%;
+    text-align:  center;
+    font-weight: normal;
 }
-.pagination a{
-    display: inline;
-    background: #080;
-    color: #FEFEFE;
-    margin: 0 .25em;
-    padding: .25em .375em;
+
+h5 {
+    font-size:   1.325em;
+    line-height: 1.325em;
+    width:       100%;
+    text-align:  center;
+    font-weight: normal;
 }
-.pagination a:hover{
-    background: #060;
-    color: #FEFEFE !important;
+
+h6 {
+    font-size:   1.25em;
+    line-height: 1.25em;
+    width:       100%;
+    text-align:  center;
+    font-weight: normal;
 }
-.pagination a:active{
-    background: #060;
-    color: #FEFEFE;
+
+a {
+    text-decoration: none;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/common/buryak/pullRequest?prid=5800482)